src/_quarto.yml

```diff
@@ -0,0 +1,255 @@
+project:
+  title: "Stan Documentation"
+  type: website
+  output-dir: _website
+  render:
+    - "index.qmd"
+    - "cmdstan-guide/overview.qmd"
+    - "functions-reference/overview.qmd"
+    - "reference-manual/overview.qmd"
+    - "stan-users-guide/overview.qmd"
+    - "cmdstan-guide/*.Rmd"
+    - "functions-reference/*.Rmd"
+    - "reference-manual/*.Rmd"
+    - "stan-users-guide/*.Rmd"
+    - "!sdlc.Rmd"
+    - "!cmdstan-guide/index.Rmd"
+    - "!functions-reference/index.Rmd"
+    - "!reference-manual/index.Rmd"
+    - "!stan-users-guide/index.Rmd"
```

Comment:
Is there a reason we now have two `index` files?

Reply:
only if we want to be able to build with bookdown - the yaml header in the index.Rmd contains a bunch of formatting info.

Reply:
I guess for an initial merge it is fine to leave it. Once we sort out the kinks I see no reason to continue to support bookdown after that